docs/en/stack/ml/anomaly-detection/ml-ad-explain.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[ml-ad-explain]]
+= Anomaly score explanation
+:keywords: {ml-init}, {stack}, {anomaly-detect}, anomaly score, typical value, \
+actual value
+:description: An explanation of how the anomaly score is calculated for \
+{ml} {anomaly-detect}.
+
+Every anomaly has an anomaly score assigned to it. That score indicates how 
+anomalous the data point is, which makes it possible to define its severity 
+compared to other anomalies. This page gives you a high-level explanation of the 
+critical factors considered for calculating anomaly scores, how the scores 
+are calculated, and how renormalization works.
+
+
+[discrete]
+[[score-impact-factors]]
+== Anomaly score impact factors
+
+Three factors can affect the initial anomaly score of a record: 
+
+* single bucket impact,
+* multi-bucket impact,
+* anomaly characteristics impact.
+
+{anomaly-jobs-cap} split the time series data into time buckets. The data within 
+a bucket is aggregated using functions. Anomaly detection is happening on the 
+bucket values.
```

Comment:
> {anomaly-jobs-cap} split the time series data into time buckets....

IMO this paragraph fits at the beginning of this section, like this:

> {anomaly-jobs-cap} split the time series data into time buckets.... bucket values. Three factors can affect the initial anomaly score of a record: ...

That way it doesn't interrupt the discussion of the meaning of each factor